elastic/kibana
# [Fleet] Fix index patterns being recreated in default space, fix migration version setting

## Summary

Closes https://github.com/elastic/kibana/issues/164243
Added a `typeMigrationVersion: '8.0.0'` to index patterns until https://github.com/elastic/kibana/issues/164454 is fixed.

Related to https://github.com/elastic/kibana/pull/161969
Feedback from kibana core team, we should always set the migration version fields, even if `migrationVersion` is not set.

It would be good to backport this to 8.9 too.

How to test locally:

- create a test space
- install system integration in test space
- restart kibana
- verify that the index patterns `logs-*`, `metrics-*` are still in test space in Data Views UI

### Checklist
